### PR TITLE
[python] remove flan-t5 python backend in recommender and add test cases

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by frankfliu — APPROVED** [review]

**Review by lanking520 — APPROVED** [review]

File: tests/integration/llm/prepare.py [hunk 1]
```diff
@@ -904,6 +904,10 @@
         "option.dtype": "fp16",
         "option.tensor_parallel_degree": 4,
         "option.max_rolling_batch_size": 4
+    },
+    "flan-t5-xl": {
+        "option.model_id": "s3://djl-llm/flan-t5-xl/",
+        "option.dtype": "fp32"
```
[the remark this answers is not in the document]
Comment (sindhuvahinis) [reply tests/integration/llm/prepare.py]: Done
